friends/src/components/FriendForm.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+
+export default class FriendForm extends Component {
+
+  constructor(props){
+    super(props);
+    this.state = {
+      name: '',
+      age: '',
+      email:''
+    }
+  }
+
+  
+
+  componentDidMount(prevProps, previousState){
+    let editFriend= this.props.editFriend;
```

Comment:
Oh nooo, use destructuring for it to be cleaner, especially for semi-long variables!
```javascript
let { editFriend } = this.props; 
```